examples/clockboard-example.rs

```diff
@@ -0,0 +1,13 @@
+// See https://github.com/zonebuilders/zonebuilder-rust/issues/23
+extern crate zonebuilder;
```

Comment:
I think you can just remove this line; `extern crate` is something that an older version of Rust needed. I think you only use it now to import macros from another crate.

Reply:
:+1: